include/mxnet/runtime/c_runtime_api.h

```diff
@@ -177,4 +178,43 @@ MXNET_DLL int MXNetObjectTypeKey2Index(const char* type_key, unsigned* out_tinde
 #ifdef __cplusplus
 }  // extern "C"
 #endif
+
```

Comment:
Can we copy over the comment too?

Reply:
thanks, only one nit: should it be MX_API_BEGIN instead of API_BEGIN?

Reply:
https://github.com/apache/incubator-mxnet/blob/master/src/c_api/c_api_common.h#L45-L55

Reply:
is this useful or should we get rid of this indirection?

Reply:
good point, removed. thanks for bringing this up

Reply:
OK, there's external dependency on this ... https://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-gpu/detail/PR-18956/7/pipeline
I will revert this change for now.